pom.xml

```diff
@@ -828,7 +828,7 @@
                     <plugin>
                         <groupId>org.owasp</groupId>
                         <artifactId>dependency-check-maven</artifactId>
-                        <version>1.4.5</version>
+                        <version>3.1.1</version>
```

Comment:
Now since we're upgrading/modifying the plugin anyway, we might as well disable the analyzers in the plugin config so that we don't have a maintenance issue between pipeline jobs and legacy jobs

i.e. [see alliance's config](https://github.com/codice/alliance/blob/d5fa4f3e7be250ea51cf39d1eafc0224d9b60c06/pom.xml#L671-L684)